src/main/java/com/stripe/param/CardUpdateOnCustomerParams.java

```diff
@@ -0,0 +1,448 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
+package com.stripe.param;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.net.ApiRequestParams;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public class CardUpdateOnCustomerParams extends ApiRequestParams {
```

Comment:
I suspect the pattern of usage is from the method call where they look at method signature, instead of going through our list of params. 
Do you think naming like `CustomerCardUpdate` and `AccountCardUpdate` would be better? We can do `CustomerCardUpdate` and `PayoutCardUpdate` but that will be more confusing.